redaxo/src/addons/cronjob/package.yml

```diff
@@ -14,3 +14,6 @@ page:
 
 requires:
     redaxo: ^5.1.0
+
+console_commands:
+    cronjob:run: rex_command_cronjob_run
```

Comment:
generell zu DX console-commands für addons entwickeln: mir ist aufgefallen dass sobald man hier was ändert in der package.yml muss man immer das addon re-installieren, damit die änderung sichtbar wird.

wenn man z.b. klassen umbenennt, oder sich mit dem key vertut (ist ja auch unüblich mit den 2x `:`) muss man wissen dass man dann ggf. re-installieren muss damit die änderungen sichtbar werden.

hab keinen vorschlag dazu, allerdings ist mir das handling "negativ aufgefallen"

Reply:
Das ist sehr merkwürdig. Hattest du probiert, ob Cache löschen auch reicht?
Wobei selbst das nicht notwendig sein sollte, da [hier](https://github.com/redaxo/redaxo/blob/10c7196611c5e0f6d6b2443d1bab3aebd737c694/redaxo/src/core/lib/packages/package.php#L251) über den Timestamp-Check der Cache neugeneriert werden sollte (falls Console oder als Admin eingeloggt).

Reply:
Ich bin in der console immer auf diesen error gelaufen
https://github.com/redaxo/redaxo/pull/1526

Reply:
Sollten wir in der console -genauso wie bei admin usern - die package.yml dateien auch immer prüfen ob aktuell? Ich glaube ja, da dort die perf. nicht so wichtig ist und es gut sein kann das man gerade ein command entwickelt

Reply:
Hmm sehe gerade dass das schon der fall sein sollte, wenn es stimmt